src/main/java/com/aws/greengrass/clientdevices/auth/iot/registry/CertificateRegistry.java

```diff
@@ -13,35 +13,26 @@
 import com.aws.greengrass.util.Utils;
 
 import java.security.NoSuchAlgorithmException;
-import java.util.Collections;
```

Comment:
High level comments:

* Ideally these registry APIs would be an interface, but our DI framework isn't flexible enough. We'll inject the registry objects into use cases. The registry itself can be constructed with whatever storage layer we need
* I'm not too concerned about an interface for the storage layer. The only thing using it is the registry itself. We can trivially refactor this later if we need other storage options.
* Don't touch CertRegistry yet. That's just creating more work for us (I'm making significant changes to it)
* ThingRegistry - skip the InMemory repo. We're not going to use it. I think you can just implement the runtime configuration storage and logic to convert from Thing to Thing configuration